Test/prover/rlimit-parse.bpl

```diff
@@ -0,0 +1,7 @@
+// RUN: %boogie /trace /proverOpt:PROVER_PATH=mocksolver.sh "%s" > "%t"
+// RUN: %OutputCheck --file-to-check "%t" "%s"
+// CHECK: resource count: -1
```

Comment:
Non-blocking nice to have: can we CHECK for the prover warning? Does that appear in stdout?

Reply:
It's supposed to get printed if you supply `/proverWarnings:1`, but that doesn't seem to be happening. I think that may be a deeper issue, now that I look at it, but perhaps one to investigate later.